hassio/src/dashboard/hassio-addons.ts

```diff
@@ -46,9 +47,24 @@ class HassioAddons extends LitElement {
                         title=${addon.name}
                         description=${addon.description}
                         ?available=${addon.available}
-                        icon=${this._computeIcon(addon)}
-                        .iconTitle=${this._computeIconTitle(addon)}
-                        .iconClass=${this._computeIconClass(addon)}
+                        ?showDot=${addon.installed !== addon.version}
+                        icon=${addon.installed !== addon.version
+                          ? "hassio:arrow-up-bold-circle"
+                          : "hassio:puzzle"}
+                        .iconTitle=${addon.state !== "started"
+                          ? "Add-on is stopped"
```

Comment:
Do we have i18n strings for this?

Reply:
Not yet, it's on my list to create PR's (one pr section, to keep it small) to have that for all stings (that make sense) under hassio/src